llmfoundry/models/layers/attention.py

```diff
@@ -411,6 +411,7 @@ def __init__(
         clip_qkv: Optional[float] = None,
         qk_ln: bool = False,
         qk_gn: bool = False,
+        fused_weights: bool = True,
```

Comment:
seems me to this should just be `fused_qkv`, right?

Reply:
sure